ihatemoney/templates/list_bills.html

```diff
@@ -86,6 +87,25 @@ <h3 class="modal-title">{{ _('Add a bill') }}</h3>
             </div>
         </div>
     </div>
+    {% if edit_bill is not none %}
+      <div id="bill-edit-form" class="modal fade show" role="dialog">
```

Comment:
what about having multiple different IDs for each of the modals here rather than sharing the same id?